bin/run_standalone.sh

```diff
@@ -30,10 +30,10 @@ if [[ "$VIRTUAL_ENV" == "" ]] ; then
   exit 1
 fi
 
-export POSTGRES_USER=snotty
-export POSTGRES_PASSWORD=MasterSlobs
-export POSTGRES_DB=postgres
-export POSTGRES_HOST=127.0.0.1
+export POSTGRES_USER=${POSTGRES_USER:-snotty}
```

Comment:
Are the (oddball) default values necessary here?

Reply:
Those are the values that @guangyee setup initially and we have continued to maintain them.

The only reason I change these lines was to allow the script to pick up the values already configured/exported in the dev-env rather than having to edit the script if it was necessary to change them.